### PR TITLE
Fix build issues

### PR DESCRIPTION
**Pull request by rahulon12**
iOS Deployment Target was not aligned with macOS, causing build issues with the Predicate macro.

### CODE REVIEW

File: CanvasPlusPlayground/PeopleManager.swift [hunk 1]
```diff
@@ -58,23 +58,26 @@ class PeopleManager {
...
     
-    func fetchAllClassesWith(userID: Int) async -> ([Course]) {
-        await fetchActiveCourses()
-        
-        var commonCourses = [Course]()
+    func fetchAllClassesWith(
+        userID: Int,
+        receivedNewCourse: @escaping (Course) -> Void = { _ in }
+    ) async {
+        let courses = await fetchActiveCourses()
```
Comment (azooz2003-bit) [inline CanvasPlusPlayground/PeopleManager.swift:79]: btw why are we fetching courses again, we already did that in course manager so we might as well pass it, fetch activeCourses here is doing another api call

File: CanvasPlusPlayground/Views/PeopleCommonView.swift [hunk 2]
```diff
@@ -19,20 +19,22 @@ struct PeopleCommonView: View {
...
 
-            ForEach(courses, id: \.id) { course in
+            ForEach(commonCourses, id: \.id) { course in
                 Text(course.name ?? "")
             }
         }
         .task {
             loading = true
-            self.courses = await peopleManager.fetchAllClassesWith(userID: user.id!)
+            await peopleManager.fetchAllClassesWith(userID: user.id!) {
```
Comment (azooz2003-bit) [inline CanvasPlusPlayground/Views/PeopleCommonView.swift:28]: we shouldve addressed this earlier, but can we prevent force unwrapping here? e.g. a guard

File: CanvasPlusPlayground/Views/PeopleCommonView.swift [hunk 3]
```diff
@@ -19,20 +19,22 @@ struct PeopleCommonView: View {
...
             }
 
-            ForEach(courses, id: \.id) { course in
+            ForEach(commonCourses, id: \.id) { course in
                 Text(course.name ?? "")
             }
         }
         .task {
             loading = true
-            self.courses = await peopleManager.fetchAllClassesWith(userID: user.id!)
```
Comment (azooz2003-bit) [inline CanvasPlusPlayground/Views/PeopleCommonView.swift:28]: correct me if im wrong, do we need to make sure commonCourses is empty

**Review by azooz2003-bit — APPROVED** [review]

**Comment by rahulon12** [comment]
Didn't realize other changes got merged in with this PR, closing this one and opening another

**Pull request closed without merging**